src/platform/forms-system/src/js/utilities/data/findDuplicateIndexes.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Find duplicates in an array of objects and return an array of indexes for the
+ *  duplicates only
+ * @param {Object} formData - Data array (ArrayField data)
+ * @param {String} dataKey - key of data of interest to find duplicates
+ * @return {Array} - Array of indexes of the duplicates only, not the first
+ *  entry
+ * @example
+ * findDuplicateIndexes([{x:'one'},{x:'two'},{x:'one'},{x:'two'}], 'x')
+ * // => [2, 3]
+ */
+export default function(formData = [], dataKey = '') {
+  if (!dataKey || formData?.length === 0) {
+    return [];
+  }
+  const list = formData.map(item => item[dataKey]?.toLowerCase() || '');
```

Comment:
```suggestion
  const list = formData.map(item => item[dataKey].toLowerCase() || '');
```
Sending a `dataKey` that doesn't point to a property of the `formData` objects feels like a bug to me. What's the use case for that behavior?

Reply:
No use case... it shouldn't be needed. I'll remove it.